src/styles/getNavigationModalCardStyles/types.ts

```diff
@@ -1,7 +1,5 @@
 import {ViewStyle} from 'react-native';
 
-type GetNavigationModalCardStylesParams = {isSmallScreenWidth: number};
-
-type GetNavigationModalCardStyles = (params: GetNavigationModalCardStylesParams) => ViewStyle;
+type GetNavigationModalCardStyles = () => ViewStyle;
```

Comment:
Why the params are being removed?

Reply:
@allroundexperts Because this type is used for typing the functions without params:
https://github.com/Expensify/App/blob/5a0b7fdb08979ea765f8a6c9e3b03e78ce61eb4a/src/styles/getNavigationModalCardStyles/index.desktop.ts#L4
https://github.com/Expensify/App/blob/5a0b7fdb08979ea765f8a6c9e3b03e78ce61eb4a/src/styles/getNavigationModalCardStyles/index.ts#L3
https://github.com/Expensify/App/blob/5a0b7fdb08979ea765f8a6c9e3b03e78ce61eb4a/src/styles/getNavigationModalCardStyles/index.website.ts#L4